tests/lib/cli-engine.js

```diff
@@ -252,6 +255,81 @@ describe("CLIEngine", function() {
             });
         });
 
+        it("should not delete code even if there are any syntax error in fixing.", function() {
```

Comment:
maybe `should not delete code if there is a syntax error after trying to autofix`